.readthedocs.yaml

```diff
@@ -3,8 +3,15 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+  jobs:
+    create_environment:
+        - asdf plugin add uv
```

Comment:
asdf?

Reply:
It's a version manager, this is from a recipe on the readthedocs site.